### PR TITLE
Add example to Animation doc

### LINKED ISSUE #17313: Adding tracks/keys to an Animation Resource via code causes errors
**Godot version:**
4f1b872

**Issue description:**
While trying to add tracks/keys to an `Animation` resource via code, this error gets printed multiple times:
```
0:00:01:0999 - Condition ' !data ' is true. returned: StringName()
----------
Type:Error
Description: 
Time: 0:00:01:0999
C Error: Condition ' !data ' is true. returned: StringName()
C Source: core/node_path.cpp:204
C Function: get_concatenated_subnames
```
But as far as I could find, they way I'm doing it is correct:
```
var animation = _animation_player.get_animation("animation")
var track_index = animation.add_track(Animation.TYPE_VALUE)
animation.track_insert_key(track_index, 0.0, var)
animation.track_set_key_value(track_index, 0, value)
```
So maybe, it could be something with Godot itself?

**EDIT:** Nope, you just have to use the `track_set_path` to define the path first. The help doc could really use a small example of setting it up.

### PR DESCRIPTION
**Pull request by YeldhamDev**
Also fixed a typo ("odes" -> "nodes").

Fixes #17313.

### CODE REVIEW

File: doc/classes/Animation.xml [hunk 1]
```diff
@@ -5,7 +5,16 @@
 	</brief_description>
 	<description>
 		An Animation resource contains data used to animate everything in the engine. Animations are divided into tracks, and each track must be linked to a node. The state of that node can be changed through time, by adding timed keys (events) to the track.
-		Animations are just data containers, and must be added to odes such as an [AnimationPlayer] or [AnimationTreePlayer] to be played back.
+		[codeblock]
+		# This creates a animation that makes the node "Enemy" move to the right by
```
Comment (akien-mga) [inline doc/classes/Animation.xml]: Should be "an" animation.

Comment (YeldhamDev) [reply doc/classes/Animation.xml]: Oops.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.0.7.

**Pull request merged**